storybook/pages/ActivityNotificationTransferOwnershipPage.qml

```diff
@@ -0,0 +1,114 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import Qt.labs.settings 1.0
+
+import mainui.activitycenter.views 1.0
+import mainui.activitycenter.stores 1.0
+
+import Storybook 1.0
+
+SplitView {
+    id: root
+
+    orientation: Qt.Vertical
+
+    Logs { id: logs }
+
+    QtObject {
+        id: notificationMock
+
+        property string id: "1"
+        property string communityId: "1"
+        property string sectionId: "1"
+        property int notificationType: 1
+        property int timestamp: Date.now()
+        property int previousTimestamp: 0
+        property bool read: false
+        property bool dismissed: false
+        property bool accepted: false
+    }
+
+    Item {
+        SplitView.fillHeight: true
+        SplitView.fillWidth: true
+
+        ActivityNotificationTransferOwnership {
+            id: notification
+
+            anchors.centerIn: parent
+            width: parent.width - 50
+            height: implicitHeight
+
+            type: ActivityNotificationTransferOwnership.OwnershipState.Pending
+            store: undefined
+            notification: notificationMock
+            communityName: communityNameText.text
+            communityColor: colorSwitch.checked ? "green" : "orange"
+
+            onFinaliseOwnershipClicked: logs.logEvent("ActivityNotificationOwnerTokenReceived::onFinaliseOwnershipClicked")
+            onNavigateToCommunityClicked: logs.logEvent("ActivityNotificationOwnerTokenReceived::onNavigateToCommunityClicked")
+        }
+
+    }
+
+    LogsAndControlsPanel {
+        SplitView.minimumHeight: 100
+        SplitView.preferredHeight: 160
+
+        logsView.logText: logs.logText
+
+        Column {
+            Row {
+                Label {
+                    text: "Community Name: "
+                }
+
+                TextInput {
+                    id: communityNameText
+
+                    text: "Doodles"
```

Comment:
I think this should be a community name

Reply:
It is storybook so I guess it does not matter. @noeliaSD

Reply:
This is `storybook` data. It's correct!